apps/emqx_bridge_pulsar/.gitignore

```diff
@@ -0,0 +1,19 @@
+.rebar3
```

Comment:
is a mistake committed?

Reply:
No, this is generated by rebar3 :thinking: 

You think it's better to remove this?